### PR TITLE
Feature/how we do

### PR DESCRIPTION
**Pull request by yagovaluchedevs**
## Description

Criando a seção Como nós fazemos

## Dev Notes

- If any developer notes are required, list them here. If not, remove the 'Notes' section.

### Steps

1. List all steps in a numbered list

## Screenshots / Videos

Add any relevant screenshots or videos. Any video should be added to the Jira ticket and linked here.

## Developer Checks

- [ ] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [ ] Tested on browser device
- [ ] Tested on Mobile device

### CODE REVIEW

**Review by jonataspinto — CHANGES_REQUESTED** [review]
Fala mano, iria colocar review em cada ponto mas é muita coisa.

- Resolver conflitos
- Usar medidas relativas
- As imagens estão na descrição da tarefa no Trello, caso não estejam me dá um alô que coloco.
- Ao invés de ter `jsx` explícito em cada texto de destaque, podemos usar o `dangernousInnerHtml` e passar uma string com toda formatação necessária igual no pr #58

**Comment by ramonxm** [comment]
Essa branch ta rolando um scroll horizontal
image

**Review by ramonxm — CHANGES_REQUESTED** [review]

File: src/mocks/listTextHowWeDo.mock.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const listTextHowWeDo = [
+  ` Nossa proposta de ensino coloca o aluno como protagonista.
```
Comment (ramonxm) [inline src/mocks/listTextHowWeDo.mock.ts:2]: Pra se assemelhar ao uso do CMS ele vem como um objeto ou array de objetos

Seria interessante deixar nesse formato: https://github.com/estartando-devs/site/blob/develop/src/mocks/DescriptionProfile.mock.ts

File: src/components/StudentCharacteristicsCard/styles.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  height: 13px;
+  background-color: #1e1e1e;
+  position: absolute;
+  top: -5px;
+  right: 2px;
+  box-shadow: 3px -3px 3px -5px white;
+  border-top-left-radius: 22px;
+  border-top-right-radius: 5px;
+  transform: rotateZ(-17deg);
+}
```
[the remark this answers is not in the document]
Comment (ramonxm) [reply src/components/StudentCharacteristicsCard/styles.ts]: ainda persisti

File: src/components/StudentCharacteristicsCard/styles.ts [hunk 3]
```diff
@@ -24,31 +25,31 @@ const TextCharacteristics = styled.p`
...
+  height: 0.812rem;
   background-color: #1e1e1e;
   position: absolute;
-  top: -4px;
+  top: -0.275rem;
   left: 0;
-  box-shadow: -4px -2px 5px -6px white;
-  border-top-right-radius: 22px;
-  border-top-left-radius: 5px;
+  box-shadow: -0.25rem -0.125rem 0.312rem -0.375rem white;
```
Comment (ramonxm) [inline src/components/StudentCharacteristicsCard/styles.ts:37]: o white tem no tema tbm

File: src/components/StudentCharacteristicsCard/styles.ts [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
+    -0.1rem -0.1rem 0.3rem -0.3rem white;
+  padding: 0.3rem 0.9rem;
   position: relative;
-  border-radius: 5px;
+  border-radius: 0.3rem;
   display: flex;
   align-items: center;
-  gap: 10px;
+  gap: 0.6rem;
   background-color: #1e1e1e;
```
Comment (ramonxm) [inline src/components/StudentCharacteristicsCard/styles.ts:16]: tem no tema

File: src/components/StudentCharacteristicsCard/styles.ts [hunk 5]
```diff
@@ -24,31 +25,31 @@ const TextCharacteristics = styled.p`
 const Division = styled.div`
   background-color: #7b7a7a;
```
Comment (ramonxm) [inline src/components/StudentCharacteristicsCard/styles.ts:26]: esse n tem, vale botar lá

**Review by ramonxm — CHANGES_REQUESTED** [review]

**Comment by ramonxm** [comment]
@yagovaluchedevs da uma olhada no files porque a resolução de conflitos trouxe muitos arquivos antigos que já foram deletados e alterados

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.64 | ✅ |
| First Contentful Paint | 3.1 s |  |
| Time to Interactive | 4.8 s |  |
| Speed Index | 3.4 s |  |
| Total Blocking Time | 470 ms |  |
| Largest Contentful Paint | 5.3 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.9 | ✅ |
| First Contentful Paint | 2.1 s |  |
| Time to Interactive | 6.2 s |  |
| Speed Index | 2.9 s |  |
| Total Blocking Time | 120 ms |  |
| Largest Contentful Paint | 2.5 s |  |
| Cumulative Layout Shift | 0.044 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.52 | ✅ |
| First Contentful Paint | 3.2 s |  |
| Time to Interactive | 6.3 s |  |
| Speed Index | 3.9 s |  |
| Total Blocking Time | 810 ms |  |
| Largest Contentful Paint | 4.9 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.92 | ✅ |
| First Contentful Paint | 1.7 s |  |
| Time to Interactive | 5.7 s |  |
| Speed Index | 2.9 s |  |
| Total Blocking Time | 260 ms |  |
| Largest Contentful Paint | 1.8 s |  |
| Cumulative Layout Shift | 0.044 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Review by ramonxm — CHANGES_REQUESTED** [review]

**Comment by ramonxm** [comment]
@yagovaluchedevs da um git pull ai, falta remover os conflitos.

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by ramonxm** [comment]
Mobile ta meio zoado 

image

File: src/components/HowWeDo/styles.ts [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+const TextHowWeDo = styled.p`
+  color: ${({ theme }) => theme.palette.design.white};
+
+  strong {
+    color: ${({ theme }) => theme.palette.design.green};
+  }
+`;
+
+const TitleHowWeDo = styled.h1`
```
Comment (ramonxm) [inline src/components/HowWeDo/styles.ts:57]: 2.25 rem é o `h2` do tema, usa o typography.

File: src/components/StudentCharacteristicsCard/index.tsx [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+const StudentCharacteristicsCard = ({
+  image,
+  textCard,
+  variation,
+}: PropsCard) => {
+  return (
+    <S.CardContainer>
+      <Image
+        src={image}
+        width="46px"
```
Comment (ramonxm) [inline src/components/StudentCharacteristicsCard/index.tsx:20]: pode ser `width={46}`

**Review by ramonxm — CHANGES_REQUESTED** [review]

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.62 | ✅ |
| First Contentful Paint | 3.0 s |  |
| Time to Interactive | 4.2 s |  |
| Speed Index | 3.8 s |  |
| Total Blocking Time | 560 ms |  |
| Largest Contentful Paint | 5.1 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.91 | ✅ |
| First Contentful Paint | 2.1 s |  |
| Time to Interactive | 5.6 s |  |
| Speed Index | 3.0 s |  |
| Total Blocking Time | 140 ms |  |
| Largest Contentful Paint | 2.5 s |  |
| Cumulative Layout Shift | 0.044 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.65 | ✅ |
| First Contentful Paint | 3.3 s |  |
| Time to Interactive | 4.9 s |  |
| Speed Index | 3.9 s |  |
| Total Blocking Time | 440 ms |  |
| Largest Contentful Paint | 4.8 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.94 | ✅ |
| First Contentful Paint | 2.0 s |  |
| Time to Interactive | 5.0 s |  |
| Speed Index | 2.8 s |  |
| Total Blocking Time | 120 ms |  |
| Largest Contentful Paint | 2.1 s |  |
| Cumulative Layout Shift | 0.044 |  |

    
</details>

File: src/components/HowWeDo/styles.ts (hunk 6 above)
Comment (jonataspinto) [reply src/components/HowWeDo/styles.ts]: VAleu

**Comment by jonataspinto** [comment]
> Mobile ta meio zoado
> 
> image

Corrigido!

**Review by jonataspinto — APPROVED** [review]

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.66 | ✅ |
| First Contentful Paint | 3.1 s |  |
| Time to Interactive | 4.9 s |  |
| Speed Index | 3.5 s |  |
| Total Blocking Time | 440 ms |  |
| Largest Contentful Paint | 4.9 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.91 | ✅ |
| First Contentful Paint | 2.1 s |  |
| Time to Interactive | 6.0 s |  |
| Speed Index | 2.8 s |  |
| Total Blocking Time | 110 ms |  |
| Largest Contentful Paint | 2.5 s |  |
| Cumulative Layout Shift | 0.044 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Review by ramonxm — APPROVED** [review]

**Pull request merged**